Cargo.toml

```diff
@@ -35,10 +35,10 @@ signatory = { version = "0.10", features = ["ed25519"] }
 signatory-dalek = "0.10"
 signatory-yubihsm = { version = "0.10", optional = true }
 subtle-encoding = "0.2"
+tempfile = "3.0.4"
```

Comment:
It looks like this is only a `[dev-dependency]`. If so, can you split it out as such?

Reply:
done